original_post_discovery.py

```diff
@@ -45,7 +45,8 @@
 MAX_PERMALINK_FETCHES = 10
 
 
-def discover(source, activity, fetch_hfeed=True, include_redirect_sources=True):
+def discover(source, activity, fetch_hfeed=True, include_redirect_sources=True,
```

Comment:
Interesting, basically everything in Flask works that way... It makes things short and sweet -- `from flask import request` is damn convenient. But I think I'd prefer to see it used more sparingly. Totally agree this would be a perfect use case for it...